### PR TITLE
OK-20821 feat:iOS support OneKey Lite V2

### PR DESCRIPTION
**Pull request by linleiqin**

### CODE REVIEW

File: packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm [hunk 1]
```diff
@@ -70,25 +21,76 @@ + (NFCISO7816APDU *)buildAPDUWith_cla:(unsigned long)cla
...
     if (JUBR_OK != rv) {
         return nil;
     }
 
-    NSString *hexStr = [NSString stringWithCString:apdu encoding:NSUTF8StringEncoding];
+    NSString *hexStr = [NSString stringWithCString:c_apdu encoding:NSUTF8StringEncoding];
     NSData *APDUData = [hexStr dataFromHexString];
-    NFCISO7816APDU *adpu = [[NFCISO7816APDU alloc] initWithData:APDUData];
-    return adpu;
+    NSLog(@"buildAPDU = %@",hexStr);
```
Comment (ByteZhang1024) [inline packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm]: 删一删，这个会泄露数据

File: packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm [hunk 2]
```diff
@@ -70,25 +21,76 @@ + (NFCISO7816APDU *)buildAPDUWith_cla:(unsigned long)cla
...
+
+    JUB_UINT16 wRet = 0;
+    JUB_CHAR_PTR pDecResp = nullptr;
+    rv = JUB_GPC_ParseAPDUResponse(c_data,&wRet, &pDecResp);
+    if (JUBR_OK != rv) {
+        return NO;
+    }
+
+    printf("Response code: %d\n", wRet);
+    printf("Decrypted response data: %s\n", pDecResp);
```
Comment (ByteZhang1024) [inline packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm]: remove

Comment (linleiqin) [reply packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm]: fixed

File: packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm (hunk 1 above)
Comment (linleiqin) [reply packages/app/ios/OneKeyWallet/NFC/OKNFCBridge.mm]: fixed

**Review by ByteZhang1024 — APPROVED** [review]

**Review by kwoktung — APPROVED** [review]

**Review by ezailWang — APPROVED** [review]

**Pull request merged**